packages/nodes-base/nodes/Discord/v2/helpers/utils.ts

```diff
@@ -271,6 +271,37 @@ export async function checkAccessToChannel(
 	checkAccessToGuild(this.getNode(), guildId, userGuilds, itemIndex);
 }
 
+export async function checkIfUserGuildMember(
+	this: IExecuteFunctions,
+	guildId: string,
+	userId: string,
+	itemIndex = 0,
+) {
+	let lastUserId;
+	let members;
+
+	while (true) {
+		members = (await discordApiRequest.call(this, 'GET', `/guilds/${guildId}/members`, undefined, {
```

Comment:
Wouldn't it be better to use this call https://discord.com/developers/docs/resources/guild#get-guild-member instead of iterating all members?

Reply:
that would definitely be better solution, thanks :+1: